### ISSUE TITLE
Deck / overview page: Start of slide tree not indicated

### ISSUE THREAD

**Issue by bwulff** [slidewiki/slidewiki-platform#1191]
On the Deck overview page there is no indication for a screen reader user where the start of the page tree (on the left) is:
A user with a screen reader can navigate the slide tree without problem. The start of the slide tree, however, is not marked by anything that a screen reader will read. At the moment this is only visually indicated. 

A quick for this would be to put a caption above the slide tree.

Another option could be to add a label element that is not visible but still read by a screen reader (added by bwulff in complete absence of knowledge about accessibility engineering on the web, so careful about this one!)
